Localization/Resources/af-ZA/winget.resw

```diff
@@ -123,7 +123,7 @@
   <data name="AdjoinedNotFoundError" xml:space="preserve">
     <value>Aangrensende vlag-alias is nie gevind nie</value>
   </data>
-  <data name="AvailableArguements" xml:space="preserve">
+  <data name="AvailableArguments" xml:space="preserve">
```

Comment:
Some projects insist on letting a localization team deal w/ changes to localizations. It isn't a big deal for me to drop changes to the `Localization/` directory.

Reply:
It isn't so much that we insist on it, but that the process of taking a localization update is destructive to your changes anyway.

In addition, src/catch2 and src/YamlCppLib/yaml-cpp are subtrees and should not be modified.

If you can drop the changes under these 3 locations, I think it will be fine to take.